website/src/queries-init.js

```diff
@@ -166,7 +166,15 @@ function getQueryOrDefault(defaultVal) {
   if (!JQ_REPO_FIELD.val()) {
     JQ_REPO_FIELD.val(defaultVal);
   }
-  return JQ_REPO_FIELD.val();
+
+  const val = JQ_REPO_FIELD.val();
+
+  const isShorthand = /^[\w\.-]+\/[\w\.-]+$/;
+  if (isShorthand.test(val)) {
+    return val;
+  } else {
+    return new URL(val).pathname;
```

Comment:
This can cause uncaught Exceptions which may confuse the user because there is nothing telling them something was wrong.
```
Uncaught TypeError: Failed to construct 'URL': Invalid URL
```
You should catch it and use `setMsg` accordingly.